lib/bat/deployment.rb

```diff
@@ -51,8 +51,13 @@ def store_manifest(content)
       @path = manifest.path
     end
 
-    def load_template(cpi)
-      template = File.expand_path("../../../templates/#{cpi}.yml.erb", __FILE__)
+    def load_template(path, cpi)
+      if (path)
+        template = File.expand_path(path, File.expand_path("../../../", __FILE__))
```

Comment:
we should just assume manifest_template_path is absolute since eventually all bats templates will live outside of this repo.